orttraining/orttraining/python/ort_trainer.py

```diff
@@ -606,6 +606,12 @@ def __init__(self, model, loss_fn, model_desc, training_optimizer_name, map_opti
                Defaults to None
             use_invertible_layernorm_grad: use invertible layernorm grad
                Defaults to False
+            _debug_onnx_model_conversion: runs a single inference step on the postprocessed ONNX model  and not postprocessed ONNX model
+               to compare with the PyTorch model. A small difference in output indicates model conversion was successful.
+               If _enable_internal_postprocess is set to False and _extra_postprocess is None, the PyTorch model will only
+               be compared with the ONNX model before postprocessing. If the output after conversion is within a reasonable
+               threshold, training continues. If it isn't, an error is produced. The PyTorch model is stored and runs on CPU.
+               Defaults to False
```

Comment:
After our last meeting, I think we should also specify what is the tolerance to error out and exit the training script (0.01)